### ISSUE TITLE
Is there any away to show label for days ?

### ISSUE THREAD

**Issue by edianogama** [fayeed/dash_chat#3]
Hi fayeed,
I need show a label for date messages like **whatsapp** from firebase.
**june1**
message1,
message2,
message3
**june2**
message1,
message2,
message3,

   ```
  List<DocumentSnapshot> items = snapshot.data.documents;
            var messages =
                items.map((i) => ChatMessage.fromJson(i.data)).toList();
```
How would i do that ?

**Comment by fayeed** [comment]
Dash chat includes this behavior by default you can check the example in example folder for more detail:

<img src="https://media.giphy.com/media/ghBkB5BHQgdhamZ2WA/giphy.gif"/>

**Comment by edianogama** [comment]
I see, I had a test yesterday at 22pm and this morning, and the label didn't show up.
I tested now at 22pm and apparently it counted at 24 hours of the previous. I do not know for sure.
I believe it may be something set by the time zone.
Is there something like that in the settings? and something related to language? 
Thank you, the other things, I managed to implement and thank you very much for your work, was perfect!
